src/components/EventCard.tsx

```diff
@@ -51,24 +51,44 @@ const EventCard = ({
     return <EventTag tag={t} />;
   });
 
+  let [regStatus, setRegStatus] = useState(true);
+
+  let regButtonStyle = regStatus ? "button-base-white px-2 my-2 mr-5" : "button-base-red px-2 my-2 mr-5";
+
+  const toggleReg = () => {
+    setRegStatus(!regStatus);
+    // POST API Call Here
+  }
+
   return (
-    <div className="flex my-8 md:my-10">
-      <EventDate startTime={startTime} status={status} />
-      <div className="flex-1 px-6 md:px-10">
-        <span className="font-sans">
-          {startTimeString + (endTime ? " to " + endTimeString : "")}
-        </span>
-        <div className="flex flex-row justify-between items-start mb-3">
-          <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
-            {name}
+      <div className="flex my-8 md:my-10">
+        <EventDate startTime={startTime} status={status} />
+        <div className="flex-1 px-6 md:px-10">
+          <span className="font-sans">
+            {startTimeString + (endTime ? " to " + endTimeString : "")}
+          </span>
+          <div className="flex flex-row justify-between items-start mb-4">
+            <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
+              {name}
+            </div>
+            <details className="relative">
+              <summary className="list-none cursor-pointer">
+                <img
+                    src={MeatballMenuSVG}
+                    alt="Menu svg"
+                    aria-label="Open Event Card details"
+                  />
+              </summary>
+              <div className="absolute -ml-40 drop-shadow-[0_12px_12px_rgba(0,0,0,0.4)] rounded-xl bg-white">
```

Comment:
Would it be possible to use one of the predefined `drop-shadow` styles, to keep shadow colors consistent?